lib/jwt_signed_request/verify.rb

```diff
@@ -68,12 +67,22 @@ def claims
       end
     end
 
-    def verified_request?
-      claims['method'].to_s.downcase == request.request_method.downcase &&
-        parsed_claims_path.path == request.path &&
-        verified_query_strings? &&
-        claims['body_sha'] == Digest::SHA256.hexdigest(request_body) &&
-        verified_headers?
+    def verify_request!
+      unless request.request_method.casecmp(claims['method'].to_s) == 0
+        raise RequestMethodVerificationFailedError, "Request failed method verification"
```

Comment:
Do you think there's any benefit to extending these error messages to show the expected vs actual? Something like `Expected GET but received POST`?

Having the expected and actual values would have been helpful when troubleshooting the author platform issues, as we were passing around the JWT token through a few services so what it was setting the values to weren't immediately apparent.